sdk/cosmos/azure-cosmos/CHANGELOG.md

```diff
@@ -22,6 +22,7 @@
 #### Other Changes
 * Shaded `MurmurHash3` of apache `commons-codec` to enable removing of the `guava` dependency - CVE-2020-8908 - See [PR 31761](https://github.com/Azure/azure-sdk-for-java/pull/31761)
 * Updated test dependency of `testng` to version 7.5 - See [PR 31761](https://github.com/Azure/azure-sdk-for-java/pull/31761)
+* Added `retyrAfterInMs` to `StoreResult` in `CosmosDiagnostics` - See [31219](https://github.com/Azure/azure-sdk-for-java/pull/31219)
```

Comment:
We should update the changelog to put it in the new unreleased version.

Reply:
Updated the changelog accordingly.